src/JobTracker/chatbot.py

```diff
@@ -26,6 +28,86 @@ def get_content(self, info):
         pass
 
 
+class Llama(ChatBot):
+    def __init__(self) -> None:
+        super().__init__()
+        self.url = LLAMA_URL
+        self.model= LLAMA_MODEL
+
+    def gen_prompt(self, info):
+        return super().gen_prompt(info)
+
+    def get_confidence(self, info):
+        return ('Rate my email as an application email on a scale of 1 to 100. return a a json \{"score": int\}  Here is the mail body: ' + info )
```

Comment:
since to be a typo in: return **a a** json \{"score": int\}  Here is the mail body: ' + info